### DISCUSSION TITLE
Blank Page When Publishing React App to Github Pages

### DISCUSSION THREAD

**Discussion by Abhijith-KOTA** [community/community#63576] [New to GitHub] [answered]
### Select Topic Area

Bug

### Body

This is the first time I am trying to deploy and a blank page is comming after deploying ,can I get some help please.

Here is the repo link: https://github.com/Abhijith-KOTA/Student-Info-System-chat-bot.git

**Comment by kmr-rohit** [comment, marked as answer]
Hi Abhijit , I reviewed your Code , the base url for Deploying app was not correctly routed. Therefore i tried to make changes in after forking you repository and after deployed successfully , i am raising a pull request kindly review and confirm if you succeeded in deployment

**Reply by Abhijith-KOTA** [reply]
Thanks for your help,but it is still showing blank page .

**Reply by kmr-rohit** [reply]
you might not deploying correctly , can you tell me which tool you are using to deploy the project ?
